src/main/java/sunny/Ui.java

```diff
@@ -0,0 +1,82 @@
+package sunny;
+
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * Handles user inputs and generate appropriate replies
+ */
+public class Ui {
+
+    // Stock messages
+    public static final String logo = "       _____   _    _  _   _  _   _  __     __  \n" +
+            "      / ____| | |  | || \\ | || \\ | | \\ \\   / /  \n" +
+            "     | (___   | |  | ||  \\| ||  \\| |  \\ \\_/ /   \n" +
+            "      \\___ \\  | |  | || . ` || . ` |   \\   /    \n" +
+            "      ____) | | |__| || |\\  || |\\  |    | |     \n" +
+            "     |_____/   \\____/ |_| \\_||_| \\_|    |_|     \n";
+    public static final String line = "     ────────────────────";
+    public static final String welcome = "     HELLO! I am Sunny:)\n     How can I help you?";
+    public static final String goodbye = "     You are leaving? Ok bye:( come back soon";
+
+    // List for tasks
+    List<Task> ls;
+
+    // File path for external storage
+    String filePath = "/Users/jerryyou/ip/taskslist.txt";
+    Storage store = new Storage(filePath);
```

Comment:
`store` might be an ambiguous name because it can be both a verb and a noun. Perhaps `storage` might be better?